src/__tests__/pages/api/auth/reset-password.test.ts

```diff
@@ -65,12 +63,10 @@ describe('Successfully change password', () => {
 
     await resetPasswordHandler(req, res);
 
-    expect(status.mock.calls[0]).toEqual([200]);
-    expect(json.mock.calls[0]).toEqual([
-      {
-        message: 'Password changed'
-      }
-    ]);
+    expect(status).toHaveBeenNthCalledWith(1, 200);
+    expect(json).toHaveBeenNthCalledWith(1, {
```

Comment:
What do all these `1`s mean?

Reply:
Normaly, when sending the requests, you usualy expect it to send 1 time and give you the respond 1 time. The 1's here can spot out if there's any abnormalities.